app/views/questions/show.html.erb

```diff
@@ -12,6 +12,11 @@
         <br>
         <div id="question-body" >
           <%= @question.body %><br>
+          <div id="delete-question">
+              <% if !current_user.nil? && current_user.id == @question.user_id %>
```

Comment:
I'm not sure why that would be error-ing with that logic change; my instinct is that something is wonky somewhere and further development will point us towards the cause.